lang/src/org/partiql/lang/domains/util.kt

```diff
@@ -16,6 +17,9 @@ fun PartiqlAst.Builder.id(name: String) =
 
 val MetaContainer.staticType: StaticTypeMeta? get() = this[StaticTypeMeta.TAG] as StaticTypeMeta?
 
+fun metaContainerOf(vararg metas: Meta): MetaContainer =
```

Comment:
Missing some kdoc. I think you can just reuse what the original function uses https://github.com/partiql/partiql-lang-kotlin/blob/03b30653480d1d315dd321e5305183d93e2efd9e/lang/src/org/partiql/lang/ast/meta.kt#L174-L175

Reply:
Will add the kdoc in the next commit.